clusterloader2/cmd/clusterloader.go

```diff
@@ -80,6 +81,7 @@ func validateClusterFlags() *errors.ErrorList {
 func initFlags() {
 	flags.StringVar(&clusterLoaderConfig.ReportDir, "report-dir", "", "Path to the directory where the reports should be saved. Default is empty, which cause reports being written to standard output.")
 	flags.BoolEnvVar(&clusterLoaderConfig.EnablePrometheusServer, "enable-prometheus-server", "ENABLE_PROMETHEUS_SERVER", false, "Whether to set-up the prometheus server in the cluster.")
+	flags.BoolEnvVar(&clusterLoaderConfig.EnableProxyPod, "enable-proxy-pod", "ENABLE_PROXY_POD", false, "Whether to set-up the proxy pod in the cluster.")
```

Comment:
I would personally say that we should call it "maintainance", and instead of having a single pod (which is always a bad idea), create a deployment with a service underneath.

@mm4tt - wdyt about "maintainance" name?

Reply:
Hmmm, what about "support" or "test-util" ?

Reply:
I changed pod to deployment. It's named "exec-pod" in "cluster-loader" namespace.